ingest/ingest_pipeline.py

```diff
@@ -161,10 +211,10 @@ def create_parser():
         description=__doc__,
         formatter_class=argparse.RawDescriptionHelpFormatter)
 
-    subparser = parser.add_subparsers()
+    subargs = parser.add_subparsers()
```

Comment:
`subargs` is a bit of a misnomer; `add_subparsers` returns an ArgumentParser object.


[Convention from the Python docs](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_subparsers) would have us name this variable `subparsers`, not `subargs`.

```suggestion
    subparsers = parser.add_subparsers()
```

(And similarly for other references to `subargs`.)

Reply:
Resolved in 00ce5f8